pkg/metrics/metrics.go

```diff
@@ -44,6 +44,18 @@ var (
 			CapacityTypeLabel,
 		},
 	)
+	NodeClaimsCreatedWithMinValuesAutoRelaxedTotal = opmetrics.NewPrometheusCounter(
```

Comment:
Is there a way to make this metric more generic? Like something about preference relaxation? This is super super specific to this change and I don't want to explode the number of metrics that we have as we start adding more config and more features

Reply:
I almost don't think that you need the metric at all -- given this setting is so specific, what's the value of surfacing this metric? If you have the annotation, someone can use something like kube-state to capture the number of nodes with relaxed preferences at any given point in time